doc/ld65.sgml

```diff
@@ -247,10 +247,11 @@ Here is a description of all of the command-line options:
   <tag><tt>-Ln</tt></tag>
 
   This option allows you to create a file that contains all global labels and
-  may be loaded into the VICE emulator using the <tt/ll/ (load label) command. You
+  may be loaded into the VICE emulator using the <tt/ll/ (load label) command
+  or into the Oricutron emulator using the <tt/sl/ (symbols load) command. You
   may use this to debug your code with VICE. Note: Older versions had some
   bugs in the label code. If you have problems, please get the latest <url
-  url="http://vice-emu.sourceforge.net/" name="VICE"> version.
+  url="http://vice-emu.sourceforge.net" name="VICE"> version.
```

Comment:
It is about the trailing slash after the FQDN (aka host name). It's optional (implicitly asking for DocumentRoot's index) when there is neither path nor query but we (at least Spiro and me :-) got used to always add it. So much that an URL without one looks wrong to us. If there is path, then the trailing slash indicates the request for default (index) document at given path. Anyway - it is not incorrect and even if we don't like this "normalisation" (as much as everything that smells node.js :-) we can't argue about it being a violation of the appropriate RFCs.